go/mcap/indexed_message_iterator.go

```diff
@@ -65,9 +66,28 @@ func (it *indexedMessageIterator) parseSummarySection() error {
 	if err != nil {
 		return fmt.Errorf("failed to seek to summary start")
 	}
+
+	summarySection := make([]byte, footer.SummaryOffsetStart-footer.SummaryStart)
```

Comment:
conformance tests caught this - SummaryOffsetStart might be 0 if there are no summary offsets, but there is a summary.

Reply:
thanks - updated